src/app/pages/property-details/property-details.component.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Landville</title>
```

Comment:
Hey @dorothyas, this section is already added in the index.html, You may have to remove it and only leave content in the body tags.

Otherwise thanks for the great work. please run `npm run lint` to see all your lint issues, fix them and then set an appropriate title for your component detail Page, You will be good to go. Need help please check out https://angular.io/guide/set-document-title

Reply:
@CryceTruly 
I have worked on this